text_localization_environment/TextLocEnv.py

```diff
@@ -1,65 +1,82 @@
 import gym
-from gym import spaces
 import random
+import numpy as np
+from gym import spaces
 from gym.utils import seeding
 from chainer.backends import cuda
 from PIL import Image, ImageDraw
 from PIL.Image import LANCZOS, MAX_IMAGE_PIXELS
-import numpy as np
-from text_localization_environment.ImageMasker import ImageMasker
+from .ImageMasker import ImageMasker
+from .transformer import create_bbox_transformer
+from .utils import scale_bboxes
 
 
 class TextLocEnv(gym.Env):
     metadata = {'render.modes': ['human', 'rgb_array', 'box']}
 
-    DURATION_PENALTY = 0.03
+    # Length of history in state
     HISTORY_LENGTH = 10
-    # ⍺: factor relative to the current box size that is used for every transformation action
-    ALPHA = 0.2
-    # η: Reward of the trigger action
+
+    # Base Reward for trigger action
     ETA = 7.0
-    # p: Probability for masking a bounding box in a new observation (applied separately to boxes 0..N-1 during premasking)
+    # Base Reward for termination action
+    ETA2 = 10.0
+    # Penalty substracted from reward
+    DURATION_PENALTY = 0.03
+
+    # Probability for masking a bounding box in a new observation (applied during premasking)
     P_MASK = 0.5
 
     def __init__(self, image_paths, true_bboxes,
         playout_episode=False, premasking=True, mode='train',
-        max_steps_per_image=200, seed=None
+        max_steps_per_image=200, seed=None, bbox_scaling=0.125,
```

Comment:
Using 1.125 rather than 0.125 would be more intuitve. Also this leaves no room for downscaling (if that's is wanted in a weird scenario). It would confuse me to set the scaling to 0.125.

Reply:
Agree but will leave it for now to not mess with the agent config